### ISSUE TITLE
Unexpected sort by name

### ISSUE THREAD

**Issue by grahamperrin** [Ulf3000/addon_manager_fix_micro#4]
1. Have two or more extensions, including _Addon Manager Fix Micro_ 1.0
2. sort by name
3. disable, or enable, anything other than _Addon Manager Fix Micro_

### Expected

- sort by name (no change)

### Actual result

- sort by last updated.

**Comment by Ulf3000** [comment]
cannot reproduce .. its also not in my code 

maybe another addon of yours reloads the tab silently so that it jumps back ?... i always test on a relatively clean dev profile and then later install in my user profile .. works alright on both

**Comment by grahamperrin** [comment]
Strange, I reproduced this in a profile with just the one extension enabled. 

I'll restart my computer and re-test.

**Comment by grahamperrin** [comment]
> I reproduced this in a profile with just the one extension enabled.

According to step 1 :-) that was a lie! Oops. 

### Saner steps to reproduce

1. have _Addon Manager Fix Micro_ 1.0 alone enabled
2. sort by name
3. reload

**Comment by Ulf3000** [comment]
yes after reloading the tab or opening a new addons manager tab , the sorters are in their initial state again (which is by last updated)  ...

my addon doesnt remember the last sorter setting ( yet!)

**Comment by Ulf3000** [comment]
yes after reloading the tab or opening a new addons manager tab , the sorters are in their initial state again (which is by last updated)  ...

my addon doesnt remember the last sorter setting ( yet!)
